doc/ref/gappkg.xml

```diff
@@ -1105,8 +1106,8 @@ part and implementation part does in general <E>not</E> allow one to actually
 <E>call</E> functions from a package when the implementation part is read.
 For example,
 in the case of a <Q>cyclic dependency</Q> as in the second example above,
-suppose that <C>B</C> provides a new function <C>f</C> or a new global record
-<C>r</C>, say, which are declared in the declaration part of <C>B</C>.
```

Comment:
Note that "say" here referred to both `f` and `B`, while your replacement only refers to `B`.

But actually, here in several other places, I'd just drop the "say" instead of inserting any replacement. Because IMHO, mathematicians are trained to read expressions involving an indefinite article, such as "*a* KIND-OF-OBJECT NAME", as saying that from here on, "NAME" refers to some arbitrary instance of KIND-OF-OBJECT. As such, the sentence is clear enough. 

But perhaps my humble opinion is wrong, so it'd be good to hear others pitch in and say how they view this.

Reply:
I think you're right there or at least I'd read it the same way.

Reply:
Agreed.